src/graph/parser.ts

```diff
@@ -289,6 +305,29 @@ export class Graph {
                         break;
                     }
 
+                    // Size fields
+                    case "width":
+                    case "height": {
+                        requiresValue()
+
+                        //search for unit
+                        let index = (value as string).search(/[A-Za-z%]/)
+                        let size = {value: 0, unit: CSSUnit.px} as Size
+                        if (index != -1) {
+                            size.value = parseFloat((value as string).substring(0, index))
+                            let unit = (value as string).substring(index)
+                            if (Object.values<string>(CSSUnit).includes(unit)) {
+                                size.unit = unit as CSSUnit
+                            }
+                        } else {
+                            size.value = parseFloat((value as string))
+                        }
+                        graphSettings[key] = size
```

Comment:
Also we should also be statically checking the non-relative units as well to prevent weird visual things with invalid units